### PR TITLE
Doc for running on Linux, requested by #75

### LINKED ISSUE #75: Make doc on how to run on Linux
Credit to `headassbtw#7274`.

### PR DESCRIPTION
**Pull request by headassbtw**
Closes #75

### CODE REVIEW

**Review by IncPlusPlus — CHANGES_REQUESTED** [review]
Thanks for your contribution! Everything looks great. I just had some really minor nitpicks. Other than that, it's good to go! 🚀

File: docs/linux/RunOnLinux.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+This document will go over how to run this application on Linux.
+
+Currently, I'm only aware of a method to run this under Lutris, so that's what I'll go over here.
+<br><br>
+It should be possible to run this under steam, but I personally haven't gotten it to work.
+## While Lutris isn't strictly required, it's recommended.
+* Head over to [the Lutris download page](https://lutris.net/downloads/) and follow the instructions for your distro.
+* Go to the [Origin page on Lutris](https://lutris.net/games/origin) and run the installer.
+![](LutrisOriginDownload.png)
+*The highlighted button should automatically open Lutris, and prompt you to install Origin, if you have Lutris installed.*
```
[the remark this answers is not in the document]
Comment (headassbtw) [reply docs/linux/RunOnLinux.md:12]: oh! i didn't notice that, as rider's md preview was too short for that to happen, nice catch!

**Comment by IncPlusPlus** [comment]
Oh, one more thing. If you could add the link to https://github.com/IncPlusPlus/titanfall2-rp/releases/latest/download/titanfall2-rp-Wine.exe somewhere in there, that'd be good. Maybe you could just turn the phrase "rich presence app" into a hyperlink to it in your instructions.

**Comment by IncPlusPlus** [comment]
Once you do that, re-request a review with the circular arrows at the top right and I'll push this through.

**Comment by IncPlusPlus** [comment]
Looks like the spacing from [here](https://github.com/IncPlusPlus/titanfall2-rp/pull/158#discussion_r742501396) didn't get applied quite right. There's also another small thing. I'll change it myself real quick.

**Review by IncPlusPlus — APPROVED** [review]
Much obliged!

**Pull request merged**